docs/content/advanced/backend/payment/how-to-create-payment-provider.md

```diff
@@ -147,26 +140,56 @@ class MyPaymentService extends AbstractPaymentService<TransactionBaseService> {
 
 ### createPayment
 
-This method is called during checkout when [Payment Sessions are initialized](https://docs.medusajs.com/api/store/#tag/Cart/operation/PostCartsCartPaymentSessions) to present payment options to the customer. It is used to allow you to make any necessary calls to the third-party provider to initialize the payment. For example, in Stripe this method is used to initialize a Payment Intent for the customer.
+This method is called during checkout when [Payment Sessions are initialized](https://docs.medusajs.com/api/store/#tag/Cart/operation/PostCartsCartPaymentSessions) to present payment options to the customer. It is used to allow you to make any necessary calls to the third-party provider to initialize the payment.
+
+For example, in Stripe this method is used to initialize a Payment Intent for the customer.
+
+The method receives a context object as a first parameter. This object is of type `PaymentContext` and has the following properties:
```

Comment:
todo: is of type Cart & PaymentContext.
This is for the backward compatibility. do you think it should be mentioned? plus it will change soon with the new payment processor API. I let you choose